internal/config/main.go

```diff
@@ -52,82 +66,58 @@ type LocalCert struct {
 	Content []byte
```

Comment:
Do you want to change this to `string` perhaps given that it's 1) base64 encoded so it doesn't matter, and 2) it is part of the snapshots so it looks a bit messy with the hexdump since it's only ASCII data?

Reply:
I would simply keep it that, as the snapshot is not to be read usually by humans